Gemfile

```diff
@@ -22,3 +22,7 @@ gem 'resque'
 gem 'nokogiri'
 gem 'memcache-client'
 gem 'sidekiq'
+gem 'rspec-core'
```

Comment:
should these be in test group? they will probably fail to be found and cause errors in prod